keel/api/v1/tasks/views.py

```diff
@@ -0,0 +1,111 @@
+
+from rest_framework.viewsets import GenericViewSet
+from rest_framework import mixins, viewsets, status as HTTP_STATUS
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from rest_framework.exceptions import ValidationError
+
+from keel.authentication.backends import JWTAuthentication
+from keel.Core.err_log import log_error
+from keel.Core.constants import GENERIC_ERROR
+from keel.tasks.models import Task, TaskComments
+
+from .serializers import ListTaskSerializer, TaskSerializer
+
+
+class ListTask(GenericViewSet):
+
+    authentication_classes = [JWTAuthentication]
+    permission_classes = (IsAuthenticated,)
+
+    def list(self, request, format = 'json'):
+
+        response = {
+            "status" : 0,
+            "message" : "Task list is successfully fetched",
+            "data" : [],
+        }
+        resp_status = HTTP_STATUS.HTTP_200_OK
+
+        user = request.user
+        user_id = user.id
+        req_data = request.GET
+        status = req_data.get("status")
+
+        if status is None or not status.isnumeric():
```

Comment:
isnumeric will throw a runtime for float or any other datatype except string
if validating please use serializers only with desired field type
avoid data validations in viewset